pep-9999.rst

```diff
@@ -0,0 +1,173 @@
+PEP: 9999
+Title: Core support for generic types
+Author: Ivan Levkivskyi <levkivskyi@gmail.com>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 03-Sep-2017
+Post-History: 09-Sep-2017
+
+
+Abstract
+========
+
+Initially PEP 484 was designed in such way that would not introduce
```

Comment:
"... way that __it__ would not introduce..."

Reply:
Thanks, fixed.